policyengine_us/variables/gov/states/co/tax/income/subtractions/military_retirement/co_military_retirement_subtraction.py

```diff
@@ -24,7 +24,7 @@ def formula(tax_unit, period, parameters):
         head = person("is_tax_unit_head", period)
         spouse = person("is_tax_unit_spouse", period)
         head_or_spouse = head | spouse
-        age_eligible = age < p.age_threshold
+        age_eligible = age <= p.age_threshold
```

Comment:
do we need to change this back as we moved the parameter back to 55?

Reply:
Yes